lib/App/Config/Chronicle.pm

```diff
@@ -4,6 +4,7 @@ package App::Config::Chronicle;
 use strict;
 use warnings;
 use Time::HiRes qw(time);
+use List::Util qw(any pairs pairmap);
```

Comment:
FYI it might be useful to include the version number of `List::Util` at which these functions appeared. You can see that in the manpage:

    pairmap
           my @list = pairmap { BLOCK } @kvlist;

           my $count = pairmap { BLOCK } @kvlist;

       Since version 1.29.

That way, if it fails it fails with a more meaningful message of

    List::Util version 1.29 required--this is only version 1.17 at /usr/share/perl/5.26/Exporter/Heavy.pm line 111.
BEGIN failed--compilation aborted.

rather than

    "pairs" is not exported by the List::Util module

Reply:
Is it sufficient to mark it in the `cpanfile` (https://github.com/CraigLawrence/perl-App-Config-Chronicle/blob/7d6f65b5df70f473987fc542517b4154db7bde18/cpanfile#L9), or do I need to here as well?